prow/jobs/test-infra/pjconfigtests.yaml

```diff
@@ -981,6 +981,47 @@ presubmits: # runs on PRs
         nodeSelector:
             dedicated: "high-cpu"
   
+  kyma-project/telemetry-manager:
+    - name: pull-telemetry-manager-pjconfigtest
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pull-telemetry-manager-pjconfigtest"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+      run_if_changed: '^.prow/|^.prow.yaml'
+      skip_report: false
+      decorate: true
+      cluster: untrusted-workload
+      max_concurrency: 10
+      branches:
+        - ^.*$
+      extra_refs:
+        - org: kyma-project
+          repo: test-infra
+          path_alias: github.com/kyma-project/test-infra
+          base_ref: main
+      spec:
+        containers:
+          - image: "europe-docker.pkg.dev/kyma-project/dev/test-infra/ginkgo:PR-6825"
+            imagePullPolicy: Always
+            securityContext:
+              privileged: false
+            command:
+              - "/ginkgo"
+            args:
+              - "-r"
+              - "/home/prow/go/src/github.com/kyma-project/test-infra/development/prow/pjconfigtests"
+            resources:
+              requests:
+                memory: 1Gi
+                cpu: 2
+        tolerations:
+          - key: dedicated
+            operator: Equal
+            value: high-cpu
+            effect: NoSchedule
+        nodeSelector:
+            dedicated: "high-cpu"
```

Comment:
Could you please explain what this job is doing?

Reply:
I don't know, it is generated and could not found any info